editor/test/integration/reload_test.clj

```diff
@@ -766,9 +766,9 @@
     (let [[workspace project] (setup-scratch world)]
       (let [node-id (project/get-resource-node project "/label/label.label")]
         (is (= "Original" (g/node-value node-id :text)))
-        (is (= [1.0 1.0 1.0] (g/node-value node-id :scale)))
         (is (= [1.0 1.0 1.0 1.0] (g/node-value node-id :color)))
         (is (= [0.0 0.0 0.0 1.0] (g/node-value node-id :outline)))
+        (is (= [0.0 0.0 0.0 1.0] (g/node-value node-id :shadow)))
```

Comment:
We no longer have `:scale` on the `LabelNode`, but I added a check for `:shadow` so the other checks wouldn't feel lonely after `:scale` left. 🙂